### PR TITLE
Revert latest PR & Add "Invert Direction" setting in the extension settings.

### PR DESCRIPTION
**Pull request by salihefee**
I apologize again for the latest PR. This reverts the changes made in the latest PR and adds an option to invert the direction.

### CODE REVIEW

File: volume_scroller@francislavoie.github.io/extension.js [hunk 1]
```diff
@@ -62,13 +68,17 @@ export default class VolumeScrollerExtension extends Extension {
   _handle_scroll(_actor, event) {
     let volume = this.sink.volume;
 
-    switch (event.get_scroll_direction()) {
+    const inverted = this._get_direction();
+    const multiplier = inverted ? -1 : 1;
+
+    const scrollDirection = event.get_scroll_direction();
+    switch (scrollDirection) {
```
Comment (francislavoie) [inline volume_scroller@francislavoie.github.io/extension.js]: Probably don't need the extra method, can simplify like this

```suggestion
    const multiplier = this.direction ? -1 : 1;
    switch (event.get_scroll_direction()) {
```

File: volume_scroller@francislavoie.github.io/extension.js [hunk 2]
```diff
@@ -104,4 +114,8 @@ export default class VolumeScrollerExtension extends Extension {
   _get_step() {
     return this.volume_max * this.volume_granularity;
   }
+
+  _get_direction() {
+    return this.direction;
+  }
```
Comment (francislavoie) [inline volume_scroller@francislavoie.github.io/extension.js]:
```suggestion
```

**Comment by francislavoie** [comment]
I think this'll still invert for both input methods since it doesn't do any input detection on the scroll event. So users might need to toggle the setting by hand if they plug in a mouse or w/e. Not perfect, but probably good enough for 99.9% of users :sweat_smile:

**Comment by salihefee** [comment]
Yes it will invert for both methods but I don't think there is an easy way of getting the source device, so this is the best I can do. Yes, it is not perfect, but at least now users have an option to invert whereas before, they would have to edit the extension source everytime they change the device :)

**Comment by francislavoie** [comment]
FYI, for next time, you shouldn't commit on your `main` branch because it causes weird conflicts. You should instead make a new branch on your own repo based on upstream's `main` and then make your commits on that. Your `main` is behind my `main` so this PR has repeated commits which isn't ideal. But it's fine, I can resolve it :+1:

Also, best practice is to put a reference to the relevant issue in the PR description (i.e. #7) to link them together

**Comment by salihefee** [comment]
I'm sorry, these two are my first pull requests and I am not experienced with git + development in general. I will keep that in mind the next time I create a PR. Thank you.

**Comment by francislavoie** [comment]
No need to be sorry :) you're doing great!

**Pull request merged**